reactive/cuda.sh

```diff
@@ -86,14 +91,12 @@ function xenial::ppc64le::install_nvidia_driver() {
 #####################################################################
 
 function trusty::x86_64::install_openblas() {
-    apt-get update -qq
     apt-get install -yqq --no-install-recommends \
         libopenblas-base \
         libopenblas-dev
 }
 
 function xenial::x86_64::install_openblas() {
-    juju-log "Not planned yet"
```

Comment:
@SaMnCo, this comment didn't make sense to me, so i removed it.  openblas installs fine on xenial/x86_64.